plugins/cost-insights/src/testUtils/providers.tsx

```diff
@@ -95,6 +95,7 @@ export const MockConfigProvider = (props: MockConfigProviderProps) => {
     products: [],
     icons: [],
     engineerCost: 0,
+    engineerThreshold: 0.5,
```

Comment:
a bit unfortunate that this is repeated here and in plugins/cost-insights/src/hooks/useConfig.tsx - maybe there's still a use for the old EngineerThreshold export for these instead of hard coding it to 0.5?

Reply:
will use the constant!